### PR TITLE
4 my new issue

### PR DESCRIPTION
**Pull request by M1ndSeeker**
task done

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,17 @@
     <title>Git test</title>
 </head>
 <body>
-    <h1>Git test page</h1>
+    <header>
```
Comment (Wladec) [inline index.html:11]: ok!

File: style.css [hunk 2]
```diff
@@ -6,4 +6,23 @@
...
+}
+
+main {
+    background-color: burlywood;
+}
+
+footer {
+    background-color: chocolate;
+    color: antiquewhite;
+    font-size: 12px;
```
Comment (Wladec) [inline style.css:27]: very nice!

**Review by Wladec — APPROVED** [review]
very good

**Comment by M1ndSeeker** [comment]
close

**Pull request merged**